server/src/controllers/questionsController.ts

```diff
@@ -313,3 +314,29 @@ export const createCommentHandler = async (req: Request, res: Response) => {
       .json({ error: "Error Adding Your Comment to the Database" });
   }
 };
+
+export const editQuestionHandler = async (req: Request, res: Response) => {
+  const questionId = Number(req.params.id);
+
+  if (!questionId) {
+    res.status(400).json({ error: "Question Id was not provided" });
+  }
+
+  const question = req.body.questionContent;
+
+  if (!question) {
+    res.status(400).json({ error: "Question was not provided" });
+  }
+
+  try {
+    const updateQuestionQuery = await editQuestion(questionId, question);
+    logger.info({
+      message: "updateQuestionQuery data",
+      value: updateQuestionQuery
+    });
+    const data = updateQuestionQuery[0];
+    res.status(200).json({ message: "success", data: data });
```

Comment:
```suggestion
    res.status(200).json(data);
```

Let's stick to the same pattern as the others, we can check on the frontend for response.ok